src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -143,10 +143,30 @@ public bool TryGetWellKnownAttribute(WellKnownAttributeFlags flag, out bool valu
             PEModuleSymbol moduleSymbol,
             PEMethodSymbol containingSymbol,
             int ordinal,
-            ParamInfo<TypeSymbol> parameter,
+            ParamInfo<TypeSymbol> parameterInfo,
+            bool isreturnParam,
```

Comment:
`isReturn...`

Reply:
Would it make sense to move the modreq check to the caller rather than passing in an `isReturnParam` argument? If so, then consider moving the modreq check for the `PEPropertySymbol` parameter case as well.

Reply:
I'd rather move it to the inner helper, rather than enumerating the modifiers twice.

---
In reply to: [165716449](https://github.com/dotnet/roslyn/pull/24524#discussion_r165716449) [](ancestors = 165716449)